dcpy/lifecycle/builds/plan.py

```diff
@@ -88,20 +88,21 @@ def plan_recipe(recipe_path: Path, version: str | None = None) -> Recipe:
         recipe.vars["VERSION_TYPE"] = recipe.version_type.value
 
     # Determine previous version
-    try:
-        previous_recipe = publishing.get_previous_version(
-            product=recipe.product, version=recipe.version
-        )
-        logger.info(
-            f"Previous version of {recipe.product}: {previous_recipe.label} ({previous_recipe})"
-        )
-        recipe.vars["VERSION_PREV"] = previous_recipe.label
-    except (
-        LookupError,
-        ValueError,
-        TypeError,
-    ) as e:  # versions not found, or don't parse correctly
-        logger.error(f"Error: {e}")
+    if "VERSION_PREV" not in recipe.vars:
```

Comment:
Just enabling manually setting the previous version (for convenience, in my case.)